.github/workflows/autoupdate.yml

```
@@ -0,0 +1,61 @@
+name: Autoupdate
+
+on:
+  schedule:
+    - cron: "0 */3 * * *"
+  workflow_dispatch:
+
+
+jobs:
+  autoupdate:
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@v2
+    - name: Install dependencies
+      run: |
+        sudo pip3 install setuptools wheel
+        sudo pip3 install yq
+        pip3 install -r scripts/requirements.txt
+    - name: Run autoupdate
+      run: |
+        git config user.email "actions@github.com"
+        git config user.name "Github Actions"
+        git fetch origin
+        
+        for file in manifests/*
+        do
+          git reset --hard master
+          ./scripts/autoupdate.py "$file"
+          
+          if [[ `git status --porcelain` ]]; then
+            echo "Things have changed, attempting to create a PR"
+            git status
+            
+            version=$(cat "$file" | yq -r ".version") # "1.5.3"
+            filename=${file##*/} # "Plugin Name.yml"
+            name=${filename%.*} # "Plugin Name"
+            branchname=$(echo "$name-$version" | sed 's/[^a-zA-Z0-9]/_/g') # "Plugin_Name-1_5_3"
```

I think a dedicated prefix would be good, as well as using a character disallowed in plugin names (`/`) to separate the version. That will support determining which existing branches target the same plugin via `cut`
```suggestion
            branchname=$(echo "auto/$name/$version" | sed 's/[^\w\d]/_/g') # "auto/Plugin_Name/1_5_3"
```

---

Very good suggestion, this is much more tidy.